dev-util/premake/premake-5.0.0_alpha16.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=${P/_/-}
+MY_PV=${PV/_/-}
+
+DESCRIPTION="A makefile generation tool"
+HOMEPAGE="https://premake.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}-core/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
First two `${PN}` could be plaintexted.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI